### PR TITLE
Work no params

### PR DESCRIPTION
**Pull request by gestchild**
Alternative to #1223

## Type
✨ Feature  

## Value
negates the need for canonical url link, by removing params from works page, while still providing back to results link

### CODE REVIEW

**Comment by jamesgorrie** [comment]
Have we conflated branches here?

**Comment by gestchild** [comment]
er, yeah looks like it - I'll sort it out

File: server/controllers/work.js [hunk 1]
```diff
@@ -25,16 +25,18 @@ function shouldUseIiif(ctx) {
 
 export const work = async(ctx, next) => {
   const id = ctx.params.id;
-  const queryString = ctx.search;
   const singleWork = await getWork(id);
   const miroId = singleWork.identifiers[0].value;
   const imgLink = imageUrlFromMiroId(miroId, shouldUseIiif(ctx));
+  const referer = ctx.request.header.referer;
```
Comment (jamesgorrie) [inline server/controllers/work.js]: We can't rely on this making it through the cache.

Comment (gestchild) [inline server/controllers/work.js]: Good point, will kill this - was just an idea

**Pull request closed without merging**